docker/bootstrap/Dockerfile.mysql57-arm64v8

```diff
@@ -1,65 +0,0 @@
-FROM debian:9 AS builder
-
-WORKDIR /opt
-#Build xtrabackup
-RUN apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-    autoconf \
-    automake \
-    bison \
-    build-essential \
-    bzr \
-    ca-certificates \
-    cmake \
-    flex \
-    libaio-dev \
-    libcurl4-gnutls-dev \
-    libev-dev \
-    libgcrypt11-dev \
-    libncurses-dev \
-    libtool \
-    mysql-client \
-    vim-common \
-    wget \
-    zlib1g-dev && \
-    wget https://github.com/percona/percona-xtrabackup/archive/percona-xtrabackup-2.4.13.tar.gz \
-        -P /opt && \
-    tar zxf /opt/percona-xtrabackup-2.4.13.tar.gz -C /opt && \
-    rm /opt/percona-xtrabackup-2.4.13.tar.gz && \
-    cd /opt/percona-xtrabackup-percona-xtrabackup-2.4.13 && \
-    mkdir bld && cd bld && \
-    cmake .. -DBUILD_CONFIG=xtrabackup_release -DWITH_MAN_PAGES=OFF \
-        -DDOWNLOAD_BOOST=1 -DWITH_BOOST=/usr/local && \
-    make -j4 && \
-    make install
-
-ARG bootstrap_version
-ARG image="vitess/bootstrap:${bootstrap_version}-common"
-
-FROM --platform=linux/arm64/v8 "${image}"
-
-# Install MySQL 5.7
-RUN add-apt-repository 'deb http://ftp.debian.org/debian sid main' && \
-    apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-    libmysqlclient-dev \
-    mysql-client-5.7 \
-    mysql-server-5.7 \
```

Comment:
These packages have long since not existed anymore on Debian since they switched to MariaDB, so this `Dockerfile` can't work atm.

Reply:
It doesn't look like this ever worked. I find no images in docker hub with this tag.